example/lib/main.dart

```diff
@@ -14,45 +13,74 @@ class MyApp extends StatefulWidget {
-  Future<void> initPlatformState() async {
-    String platformVersion;
-    // Platform messages may fail, so we use a try/catch PlatformException.
-    // We also handle the message potentially returning null.
-    try {
-      platformVersion =
-          await AndroidLocatorPlugin.platformVersion ?? 'Unknown platform version';
-    } on PlatformException {
-      platformVersion = 'Failed to get platform version.';
-    }
-
-    // If the widget was removed from the tree while the asynchronous platform
-    // message was in flight, we want to discard the reply rather than calling
-    // setState to update our non-existent appearance.
-    if (!mounted) return;
-
-    setState(() {
-      _platformVersion = platformVersion;
-    });
   }
 
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      debugShowCheckedModeBanner: false,
       home: Scaffold(
         appBar: AppBar(
-          title: const Text('Plugin example app'),
+          title: const Text(
+            ExampleConstants.appBarTitle,
+          ),
         ),
         body: Center(
-          child: Text('Running on: $_platformVersion\n'),
+          child: Column(
+            children: [
+              Padding(
+                padding: const EdgeInsets.only(
+                  top: ExampleConstants.edgeInsetsTop,
+                ),
+                child: PluginButton(
+                  text: ExampleConstants.checkPermission,
+                  onPressed: () {
+                    AndroidLocatorPlugin.checkPermission;
+                  },
+                ),
+              ),
+              StreamBuilder(
+                stream: AndroidLocatorPlugin.accessStream,
+                builder: (
+                  BuildContext context,
+                  AsyncSnapshot<dynamic> snapshot,
+                ) {
+                  return snapshot.hasData
+                      ? Text(
+                          snapshot.data.toString(),
+                        )
+                      : Text(
+                          ExampleConstants.permissionInfo,
+                        );
+                },
+              ),
+              Padding(
+                padding: const EdgeInsets.only(
+                  top: ExampleConstants.edgeInsetsTop,
+                ),
+                child: PluginButton(
+                  text: ExampleConstants.requestPermission,
+                  onPressed: () {
+                    AndroidLocatorPlugin.requestPermission;
+                  },
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(
+                  top: ExampleConstants.edgeInsetsTop,
+                ),
+                child: PluginButton(
+                  text: ExampleConstants.initializePlugin,
+                  onPressed: () {
+                    AndroidLocatorPlugin.initializePlugin;
+                  },
+                ),
+              ),
```

Comment:
I think a better approach to this is to create a Widget that holds Padding and PlugginButton

Reply:
Done. Now the padding is applied when PlugginButton is created